apps/src/templates/sectionAssessments/MultipleChoiceSurveyOverviewTable.jsx

```diff
@@ -51,19 +54,22 @@ const answerColumnsFormatter = (
 ) => {
   const cell = rowData.answers[columnIndex - 1];
 
-  let percentValue = 0;
+  // Default value for questions that don't include this column
+  let percentValue = -1;
 
   if (property === NOT_ANSWERED) {
     percentValue = rowData.notAnswered;
-  } else {
```

Comment:
nice bit of cleanup here

Reply:
hah, thanks! that one wasn't as simple as it looks. it turns out the `PercentAnsweredCell` defaults to -1 if `percentValue` is undefined, so we were relying on that line unsetting the value if we don't have a cell. imo much clearer to explicitly pass -1 in that case, hence my comment above.